lib/command/src/Obelisk/Command/Thunk.hs

```diff
@@ -93,6 +93,11 @@ data ThunkSource
    | ThunkSource_Git GitSource
    deriving (Show, Eq, Ord)
 
+thunkSourceToGitSource :: ThunkSource -> GitSource
+thunkSourceToGitSource = \case
+  ThunkSource_GitHub s -> forgetGithub False s
+  ThunkSource_Git s -> s
```

Comment:
I feel like I've seen this before. Is it already written somewhere else?

Reply:
I found a few similar cases. Switched them to use this function